src/main/java/eu/europeana/metis/sandbox/dto/debias/DeBiasReportDto.java

```diff
@@ -2,14 +2,18 @@
 
 import com.fasterxml.jackson.annotation.JsonFormat;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import eu.europeana.metis.sandbox.service.workflow.DeBiasProcessServiceImpl;
+import eu.europeana.metis.sandbox.service.workflow.DeBiasProcessServiceImpl.DeBiasReportRow;
 import io.swagger.annotations.ApiModel;
 import java.time.ZonedDateTime;
+import java.util.Collections;
+import java.util.List;
 
 /**
  * The type Detection info dto.
  */
-@ApiModel(DetectionInfoDto.SWAGGER_MODEL_NAME)
-public class DetectionInfoDto {
+@ApiModel(DeBiasReportDto.SWAGGER_MODEL_NAME)
+public class DeBiasReportDto {
```

Comment:
Is the DeBias state also available in another call? It seems inefficient to download the whole report repeatedly just to know whether the report is ready for downloading.

Reply:
correct. I'll split this functionality.